### DISCUSSION TITLE
How can I build a Vite, TS, HTML framework Storybook so I end up with a single CSS and JS file?

### DISCUSSION THREAD

**Discussion by dibs** [storybookjs/storybook#29191] [Help]
### Summary

I'm trying to build for a Drupal project which can see the story twig templates and I want a single output file of css and js that contains every story in one file. When I run `$ npm run build-storybook` I get a storybook-static folder filled with files for each story but is there an option to output the css and js as monolithic files?

### Additional information

Here's my main.ts
```
import type { StorybookConfig } from "@storybook/html-vite"

const config: StorybookConfig = {
  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
  addons: [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/addon-knobs",
    "@storybook/addon-interactions",
    "@storybook/addon-a11y",
  ],
  framework: {
    name: "@storybook/html-vite",
    options: {},
  },
  docs: {
    autodocs: "tag",
  },
  staticDirs: ['../public', '../static'],
}
export default config
```

**Comment by twalkerspire** [comment]
i am wondering this same question
